docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md

```diff
@@ -9,32 +9,32 @@ To help make setting up Expensify Chat for your event and your attendees super s
 
 ## Who you are
 
```

Comment:
Does Jekyll cut down on extra spaces? This doc is formatted pretty differently to the other one, with a lot more whitespace before/after section headers, etc.

Reply:
that's a good question, i can't remember. I'll spin these up locally and get screenshots!